rollup.config.mjs

```diff
@@ -0,0 +1,67 @@
+import commonjs from "@rollup/plugin-commonjs";
+import nodeResolve from "@rollup/plugin-node-resolve";
+import external from "rollup-plugin-peer-deps-external";
+import terser from "@rollup/plugin-terser";
+import typescript from "rollup-plugin-typescript2";
+
+const input = "src/index.ts";
+
+const plugins = [
+  external({
+    includeDependencies: true,
+  }),
+  typescript({
+    clean: true,
+    exclude: ["**/__tests__", "**/*.test.ts", "**/stories/**/*"],
+  }),
+  commonjs({
+    include: /\/node_modules\//,
+  }),
+  nodeResolve(),
+  terser({
+    output: { comments: false },
+    compress: {
+      drop_console: false, // has to be changed back
```

Comment:
```suggestion
      drop_console: false,
```

I think it makes sense to keep console statements for this package since we want to log when a secret was loaded.